examples/widgets/buttons.py

```diff
@@ -2,7 +2,7 @@
 =======
 Buttons
 =======
-
+An pretty cool example to show how to use buttons widget.The program shows a simple GUI with a sine wave changing its frequency.The next and previous button widget used here helps shows the wave with new frequency.
```

Comment:
An pretty cool? You should not have had the first line, a small description should be there. Don't use the words like "an example to show etc"

Reply:
Sorry it was 'A' . And yes will remove the first line.